### PR TITLE
[Integration guide] Add document

### PR DESCRIPTION
**Pull request by atorin**

### CODE REVIEW

File: content/developers/en/integration.md [hunk 1]
```diff
@@ -7,6 +7,224 @@ meta:
...
+### Receiving Orders and Updating Their Status
+
+Registering an active callback is the recommended way to receive new orders. As an alternative, use a passive callback to fetch new events at regular intervals.
+
+When you process the payload, bear in mind the following rules:
+
+- Support E.164 encoding for phone numbers.
+- Do not reject orders if prices or ref codes are wrong.
+
+For other tips and suggestions, check our [Integration Sheet](https://docs.google.com/spreadsheets/d/1df-QRlD9h8M58bpFoFaCEzU5pbmYSeHXOLqIVip9-5s/edit?usp=sharing).
```
Comment (antmonnier) [inline content/developers/en/integration.md:175]: Every time we refer to the Integration Sheet, I think we should designated the Tab in which they will find the information.
Some developers may not even realize the sheet contains several tabs.

**Review by antmonnier — CHANGES_REQUESTED** [review]
Hi Alberto. Don't let the number of suggestions fool you: you did a great job! The amount of feedback just reflects the importance of this page.

Last bits of feedback: this page should be renamed "Integration guide" (title + filename & path). The main /developers page should be updated with the new name, and it should describe this guide as an "Integration checklist and best practices to build your integration" (or sth similar)

**Comment by janainawittner** [comment]
@atorin, @antmonnier, I removed myself as a reviewer. When you guys are ready, if you would like me to go over it, just ping me again :-)

File: content/developers/en/integration.md [hunk 2]
```diff
@@ -7,6 +7,224 @@ meta:
   description:
 ---
 
-We are working on this chapter.
+## Overview
 
-Meanwhile, you can check the [API](/developers/api/general-concepts) or <ContactFormToggle text="contact us" />.
+This document provides all the information you need to integrate your solution with HubRise.
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/developers/en/integration.md:12]: You're right! :)

File: content/developers/en/integration.md [hunk 3]
```diff
@@ -7,6 +7,224 @@ meta:
...
+
+Some general best practices are applicable to most integrations. But we know that every solution is different, so we will give specific suggestions for different types of case scenarios (online ordering platforms, EPOS solution, etc).
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/developers/en/integration.md:14]: I agree that this information is a bit hidden with this format, but I'm not sure about a bullet list with only 2 items. 
How about we rely on a hyperlink to make this info stand out? We can add another for "general best practices", so we get a mini TOC inside the sentence...

[the remark this answers is not in the document]
Comment (antmonnier) [reply content/developers/en/integration.md:14]: I'm happy with any solution, as long as it highlights the structure of the document (even for a reader scanning through)

File: content/developers/en/integration.md [hunk 4]
```diff
@@ -7,6 +7,224 @@ meta:
...
+
+## General Best Practices
+
+When you start developing your integration, some smart choices at the beginning can save you time and effort along the way. Here are a few suggestions.
+
+### Create a Support Alias for HubRise
+
+When you create a HubRise account for your integration, use an email alias such as <support@your-solution.com>, with an email forward to your support team. In this way, all your support team will be able to access the HubRise logs for debugging purposes.
+
+You can then provide access to your HubRise account to as many individual users as you want (such as <alice@your-solution.com> or <bob@your-solution.com>).
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/developers/en/integration.md:46]: Yes, my bad! There was a misunderstanding on my part.

File: content/developers/en/integration.md [hunk 5]
```diff
@@ -7,6 +7,224 @@ meta:
...
+
+When your integration is ready, we will review it with you based on our [Integration Sheet](https://docs.google.com/spreadsheets/d/1df-QRlD9h8M58bpFoFaCEzU5pbmYSeHXOLqIVip9-5s/edit?usp=sharing).
+
+In this way, we will know exactly what your integration can and cannot do, and we will be able to document it thoroughly.
+
+### Documenting the Integration
+
+Once your integration is completed and assessed, we will document it and include it in our [Integrated Apps](/apps) page, as we did in the past with other integrations (like dotdigital, PAR PixelPoint, LivePepper, Zelty, to name a few).
+
+This is good for your SEO and ours. It also helps maximising transparency by showing exactly what your integration does.
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/developers/en/integration.md:36]: OK! Used a bullet list instead, but I can revert back to a single sentence if that's too much.

File: content/developers/en/integration.md [hunk 6]
```diff
@@ -7,6 +7,224 @@ meta:
...
+- Include `catalog.read` or `catalog.write` permissions only if your solution handles catalogs.
+- Register an active callback with the `"order": ["update"]` event.
+- See [Connection Workflow](#connection-workflow) for other best practices on managing your solution's connection.
+
+### Pushing Customers to HubRise
+
+If users can create an account on your solution, you should push their details to HubRise as soon as they register.
+More generally, every time users update their details, you should update them on HubRise, as well.
+
+You should also anonymise customers immediately if they request so, or if they have been inactive for a certain period of time (for example, 3 years). This is a legal requirement in the EU under GDPR regulations. For more details, see [Anonymize customer](https://www.hubrise.com/developers/api/customer-management/#anonymize-customer).
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/developers/en/integration.md:107]: OK, I agree.

File: content/developers/en/integration.md [hunk 7]
```diff
@@ -7,6 +7,224 @@ meta:
...
+
+- Products and skus (names and ref codes).
+- Options (names and ref codes). Multiple identical options should be encoded as separate individual objects.
+- Deals (names and ref codes) and deal line labels.
+- Discounts (names and ref codes).
+- Charges (names and ref codes).
+- Payments (names and ref codes). You should always pass at least one payment.
+- Service type (names and ref codes).
+- Customer id, if the order is associated with a customer.
+- Times, encoded in the timezone of the store. Expected delivery times should always be in the future or `null`.
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/developers/en/integration.md:230]: Good idea!

**Comment by antmonnier** [comment]
In OLO, we should probably separate "Pushing orders" in two sections: "Pushing orders" and "Receiving status updates"

In the new section, please mention that the OLO should monitor order statuses and raise an alert (email to the restaurant managers and/or to the customer) if an order is still in `new` status after a delay of 5 minutes (or so)

**Pull request merged**